userspace/libsinsp/sinsp.h

```diff
@@ -749,8 +749,8 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 	void unset_eventmask(uint32_t event_id);
 
 	/*!
-	  \brief When reading events from a trace file, this function returns the
-	   read progress as a number between 0 and 100.
+	        \brief When reading events from a trace file, this function returns the
+		       read progress as a number between 0 and 100.
```

Comment:
Hard to say whether this improves or breaks the indentation based on the github diff, but if the only changes to this file are in whitespace, I'd consider dropping them to save a few tons of CO2 in unnecessary rebuilds :) (this file is included absolutely everywhere)